workshop/exercises/pygeoapi.config.yml

```diff
@@ -184,6 +184,37 @@ resources:
               data: tests/data/ne_110m_admin_0_countries.sqlite
               id_field: ogc_fid
               table: ne_110m_admin_0_countries
+    airports:
```

Comment:
Please move this block to the ["Exercise 5" section (before the wms-facade) ](https://github.com/geopython/diving-into-pygeoapi/blob/c62bbf60d68378a9659ce007e658ff70f0e01c98/workshop/exercises/pygeoapi.config.yml#L391)and leave it commented out.